app/main.py

```
@@ -1,42 +1,61 @@
-from fastapi import FastAPI, HTTPException
-from schemas.product import Product
-from schemas.category import Category
-from utils import load_data
-from typing import List, Dict
+from fastapi import FastAPI
+from database import Database
+from dotenv import load_dotenv
+from models.Base import base
+from models import models
```

Ruff (0.8.2)

`models.models` imported but unused

Remove unused import: `models.models`

(F401)